docs/src/snap/howto/refresh-certs.md

```diff
@@ -0,0 +1,98 @@
+# Refreshing Kubernetes Certificates
+
+Keeping your {{product}} cluster secure and functional requires
+regularly refreshing its certificates. Certificates in Kubernetes ensure
+secure communication between the various components of the cluster. If
+these certificates expire, it can lead to communication failures, disrupted
+services, and potential security risks. This how-to will walk you through
+the steps to refresh the certificates for both control plane and worker
+nodes in your {{product}} cluster.
+
+## What you will need
+- A running {{product}} cluster
+
+```{note} To refresh the certificates in your cluster, make sure it was
+initially set up with self-signed certificates or with your own CA certificates
+and keys during the bootstrap process.
+```
+
+### Refreshing Control Plane Node Certificates
+To refresh the certificates on control plane nodes, follow these steps:
+
+1. Access each control plane node in your cluster.
+2. Run the `refresh-certs` command:
+
+```bash
+sudo k8s refresh-certs --expires-in 1y --extra-sans mynode.local
+```
+
+This command refreshes the certificates for the control plane node, adding an
+extra Subject Alternative Name (SAN) to the certificate. The `--expires-in`
+flag sets the certificate's validity duration, which can be specified in years,
+months, days, or any unit accepted by the [ParseDuration][] function in Go.
+
+```{note} Ensure that you provide the same SANs that were used when the cluster
+was bootstrapped. If you don't, the control plane may fail to communicate with
+other nodes in the cluster.
+```
+3. The cluster will automatically update the certificates in the control plane
+node and restart the necessary services. The command output will show the new
+expiration date:
+```bash
+Certificates have been successfully refreshed, and will expire at 2025-08-27 21:00:00 +0000 UTC.
+```
+
+### Refreshing Worker Node Certificates
+
+To refresh the certificates on worker nodes, follow these steps:
+
+1. Access each worker node in your cluster.
+
+2. Run the `refresh-certs` command:
+```bash
+sudo k8s refresh-certs --expires-in 10y --timeout 10m
+```
+This command refreshes the certificates for the worker node. The `--expires-in`
+flag specifies the certificate's validity period. As mentioned in the control
+plane section, this duration can be set using any unit accepted by the
+[ParseDuration][] function in Go, in addition to years, months, or days.
+
+3. During this process, multiple Certificate Signing Requests (CSRs) are created.
+The command output will guide you on how to approve the CSRs on any control plane
+node in the cluster.
+```bash
+root@w-1:~# k8s refresh-certs --expires-in 10y --timeout 10s
```

Comment:
nit: commands and output should always be separated so that users can easily copy the commands. For the same reason, I'd remove the `root@w-1:#` prefix

Reply:
Sure, sounds good